jsonschema/cli.py

```diff
@@ -1,6 +1,10 @@
 """
 The ``jsonschema`` command line.
 """
+
```

Comment:
Could you possibly remove these heading-style comments? I don't find they add much.

Reply:
Ok everyone has its own conventions :) I’ll remove that.